### PR TITLE
Hw3

### PR DESCRIPTION
**Pull request by Konstantin711**
All tasks for Lesson 2 are finished

### CODE REVIEW

File: hw_2_task_1.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# 1
+# You have 2 companies with people. One of the companies, let it be Eleks,
+# was taken over by Toshiba. Show it in code. Keep in mind that people
+# with the same name can be in both companies
+
+eleks = ['Johny D', 'Rich Wood', 'Carla Aster', 'Oliver Good']
+toshiba = ['Marry Moore', 'Tim Cook', 'Oliver Good']
+toshiba.extend(eleks)
+take_over = toshiba
```
Comment (bohdandovbsyh) [inline hw_2_task_1.py]: you do not need this variable.
also eleks still have employees

File: hw_2_task_3.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+all_guests = []
+
+while omnivore:
+    guest = omnivore.pop()
+    all_guests.append(guest)
+
+    if not omnivore:
+        for vegan in vegetarian:
+            all_guests.append(vegan)
```
Comment (bohdandovbsyh) [inline hw_2_task_3.py]: please do not use loops for this HW

File: hw_2_task_4.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# 4
+# You have a group of guests for the VIP box. The seats in the VIP box
+# are all occupied by these guests. There is a group of guests in the
+# common room and there are still places in it. Display 2 groups of guests in code.
+
+vip_guests = ['Jenny Tyler', 'Sad One', 'Frank Sinatra']
```
Comment (bohdandovbsyh) [inline hw_2_task_4.py]: please make it tuple

File: hw_2_task_5.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+
+non_uniq = ['Jenny Tyler', 'Sad One', 'Frank Sinatra', 'Sad One', 'Frank Sinatra', 'Jenny Tyler', 'Sad One', 'Frank Sinatra']
+
+flag = 0
+uniq_names = []
+
+for name in non_uniq:
+    if name not in uniq_names:
+        uniq_names.append(name)
```
Comment (bohdandovbsyh) [inline hw_2_task_5.py]: you can do this task without loops.

File: hw_2_task_1.py (hunk 1 above)
Comment (Konstantin711) [reply hw_2_task_1.py]: Fixed

File: hw_2_task_3.py (hunk 2 above)
Comment (Konstantin711) [reply hw_2_task_3.py]: Loops were removed

File: hw_2_task_4.py (hunk 3 above)
Comment (Konstantin711) [reply hw_2_task_4.py]: Fixed

File: hw_2_task_5.py (hunk 4 above)
Comment (Konstantin711) [reply hw_2_task_5.py]: Fixed

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**